libraries/rush-sdk/src/index.ts

```diff
@@ -61,8 +69,53 @@ if (rushLibModule === undefined) {
 
 // SCENARIO 3:  A tool or script depends on "rush-sdk", and is meant to be used inside a monorepo folder.
 // In this case, we can use install-run-rush.js to obtain the appropriate rush-lib version for the monorepo.
-//
-// NOT IMPLEMENTED YET
+if (rushLibModule === undefined) {
+  try {
+    const rushJsonPath: string | undefined = findUp.sync('rush.json', {
```

Comment:
This functionality already exists as `RushConfiguration.tryFindRushJsonLocation`. Can we just reuse that?

Reply:
Unfortunately no, since the task of `rush-sdk` is to load `rush-lib`, it can not reuse logic inside rush-lib. but i can remove the dependency of `find-up` and copy paste the logic to `rush-sdk`. hopes it makes thing better.